source/common/upstream/cds_api_impl.cc

```diff
@@ -71,56 +59,8 @@ void CdsApiImpl::onConfigUpdate(const std::vector<Config::DecodedResourceRef>& r
 void CdsApiImpl::onConfigUpdate(const std::vector<Config::DecodedResourceRef>& added_resources,
                                 const Protobuf::RepeatedPtrField<std::string>& removed_resources,
                                 const std::string& system_version_info) {
-  Config::ScopedResume maybe_resume_eds;
```

Comment:
To aid review, can you split this pure refactor out into a separate PR? Basically just make the helper class and move the code, and then we can get back to this PR?

Reply:
Please see https://github.com/envoyproxy/envoy/pull/15806.